beacon-chain/operations/attestations/forkchoice/forkchoice.go

```diff
@@ -0,0 +1,89 @@
+package forkchoice
+
+import (
+	"sync"
+
+	"github.com/pkg/errors"
+	ethpb "github.com/prysmaticlabs/prysm/v5/proto/prysm/v1alpha1"
+	"github.com/prysmaticlabs/prysm/v5/proto/prysm/v1alpha1/attestation"
+)
+
+// Attestations --
+type Attestations struct {
```

Comment:
I don't get this. What makes it "fork choice"? Seems like generic attestation cache

Reply:
I renamed the package to `attmap`